k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml

```diff
@@ -60,15 +60,6 @@
     "sha256:05ac4b4aef14d4a594afc8e6a63076197debfee2a7b8c191f65c7d653beee0e0": ["1.15-alpine"]
     "sha256:503b7abb89e57383eba61cc8a9cb0b495ea575c516108f7d972a6ff6e1ab3c9b": ["1.14-1", "1.14-1-copy"]
     "sha256:6e029eca4e40d373819f5fd710c487a7f002d6ae70de85989348c18eef7f22f5": ["1.15-1"]
-- name: node-perf/npb-ep
-  dmap:
-    "sha256:ac7a746f351635663abb0c240c0af71b229d1e321e478664c7816de4f4176818": ["1.1"]
-- name: node-perf/npb-is
-  dmap:
-    "sha256:4d0c0cef373fba0752721552f8d7a478156c255c8dbf90522165784e790f1ab7": ["1.1"]
-- name: node-perf/tf-wide-deep
-  dmap:
-    "sha256:d5d5822ef70f81db66c1271662e1b9d4556fb267ac7ae09dee5d91aa10736431": ["1.1"]
```

Comment:
I'm not sure these are directly related to the app in question, do you know where these are used?

/cc @listx @justaugustus 
What's the policy on deleting promotion manifests? I feel like we should be leaving these along and treat them as append-only for provenance (wait, _where_ did this image in k8s.gcr.io come from again?), but maybe that's not necessary given the promoter effectively makes k8s.gcr.io append-only

Reply:
Currently we don't have a policy on deleting manifests. I agree that leaving them alone is a better idea as it makes it easier to search the intent (just grep through `main` branch for the digest, instead of needing to do `git log -S <digest>` on top of grepping).